api/src/pcapi/core/users/api.py

```diff
@@ -841,7 +841,10 @@ def create_pro_user(pro_user: ImportUserFromCsvModel | ProUserCreationBodyV2Mode
 
     if settings.IS_INTEGRATION:
         new_pro_user.add_beneficiary_role()
-        new_pro_user.validatedBirthDate = new_pro_user.dateOfBirth
+        new_pro_user.dateOfBirth = datetime.datetime.utcnow() - datetime.timedelta(
+            days=366 * 18
+        )  # Some weeks after 18th birthday
+        new_pro_user.validatedBirthDate = new_pro_user.dateOfBirth.date()
```

Comment:
<!-- Nitpicks are small, trivial, but necessary changes. Distinguishing nitpick comments significantly helps direct the reader's attention to comments requiring more involvement.  -->
**🤓 nitpick:** ‏Le commentaire est un peu flou, c'est pas avant son anniversaire la date
```suggestion
        eighteen_years_ago = datetime.datetime.utcnow() - datetime.timedelta(
            days=366 * 18
        )
        new_pro_user.dateOfBirth = eighteen_years_ago
        new_pro_user.validatedBirthDate = new_pro_user.dateOfBirth.date()